sdk/servicebus/service-bus/src/sender.ts

```diff
@@ -247,13 +247,12 @@ export class SenderImpl implements Sender {
       return this._sender.sendBatch(batch, options);
     } else if (isServiceBusMessageBatch(messageOrMessagesOrBatch)) {
       return this._sender.sendBatch(messageOrMessagesOrBatch, options);
+    } else if (isServiceBusMessage(messageOrMessagesOrBatch)) {
+      return this._sender.send(messageOrMessagesOrBatch, options);
     } else {
-      throwTypeErrorIfParameterMissing(
-        this._context.namespace.connectionId,
-        "message, messages or messageBatch",
-        messageOrMessagesOrBatch
+      throw new TypeError(
+        "Invalid type for message. Must be a ServiceBusMessage, an array of ServiceBusMessage or a ServiceBusMessageBatch"
```

Comment:
What about the case when user passes in an array, but the array items are strings?
Also, what if a string is passed to `tryAdd()`?

cc @richardpark-msft

Reply:
Makes sense to go down the rabbit hole a bit further :) I'll file an issue for it.

Reply:
Filed #9487